src/components/ChartCard/ChartCardHeader-Ring.vue

```diff
@@ -115,7 +115,7 @@ export default {
 
       <div class="text-h3 font-weight-bold d-flex align-start justify-end">
         {{ chartOverlayMain
-        }}<span class="title font-weight-black">{{ chartOverlayUnits }}</span>
```

Comment:
For my own knowledge, how would I know if I'm breaking something here? My concern was we might be using styling in which `title` had expected styling which `text-h6` would not.

Reply:
Great question! If you're running the development server, deprecated classes like this will get caught by the prettier rules coming from Vuetify; you'd see an overlay on the application that tells you which classes are incorrect/deprecated. In this case, these components are part of our library but aren't being used anywhere at the moment and so don't get parsed when the dev server is running (hence these didn't get caught when I updated all the classes before). If you were to import either of these components this would have shown up immediately.

Reply:
this was from an overarching change in how Vuetify names their text classes, Nicholas did a sweep of class names a week or so ago but may have just missed this one.